x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_form/relevance_tuning_form.scss

```
@@ -0,0 +1,9 @@
+.relevanceTuningForm {
+  .relevanceTuningForm__item {
+    width: 100%;
```

Super small spacing suggestion (adds a bit more negative space away from the EUI arrow) here:

```suggestion
    width: 100%;
    margin-left: $euiSizeS;
```

---

https://github.com/elastic/kibana/pull/90621/commits/805f5afc952c6ab0dc297308aa9b813c8629aeaf